### PR TITLE
[Wait for #1274] [Test/models] Add addition test

### PR DESCRIPTION
**Pull request by zhoonit**
**[Pending commit: #1274]**

- [Test/models] Add addition test

```
This patch adds addition test, while add support for reading multiple
inputs and outputs inside `unittest_nntrainer_models.cpp`

**Self evaluation:**
1. Build test: [X]Passed [ ]Failed [ ]Skipped
2. Run test: [X]Passed [ ]Failed [ ]Skipped

Signed-off-by: Jihoon Lee <jhoon.it.lee@samsung.com>

```

### CODE REVIEW

**Review by jijoongmoon — APPROVED** [review]
LGTM

File: test/unittest/unittest_nntrainer_models.cpp [hunk 1]
```diff
@@ -266,18 +289,29 @@ class GraphWatcher {
 };
 
 void NodeWatcher::read(std::ifstream &in) {
-  expected_output.read(in);
-  expected_dx.read(in);
+  // log prints are commented on purpose
+  // std::cout << "[=======" << node->getName() << "==========]\n";
+
+  auto read_ = [&in](auto &target) {
+    // std::cout << target.getDim();
```
Comment (kparichay) [inline test/unittest/unittest_nntrainer_models.cpp:296]: cleanup couts?

File: test/unittest/unittest_nntrainer_models.cpp [hunk 2]
```diff
@@ -1184,7 +1225,7 @@ INSTANTIATE_TEST_CASE_P(
     mkModelTc(preprocess_flip_validate, "3:1:1:10", 10),
 
     /**< Addition test */
-    mkModelTc(addition_resnet_like_validate, "3:1:1:10", 10),
+    mkModelTc(addition_resnet_like, "3:1:1:10", 10),
```
Comment (kparichay) [inline test/unittest/unittest_nntrainer_models.cpp:1228]: how about update the dimension to bigger dimensions where each dimension is unique and not 1?

**Review by lhs8928 — APPROVED** [review]
LGTM

File: test/unittest/unittest_nntrainer_models.cpp (hunk 1 above)
Comment (zhoonit) [reply test/unittest/unittest_nntrainer_models.cpp:296]: Intended, this is too descriptive to fit in ml_logd bug needed, so commented

File: test/unittest/unittest_nntrainer_models.cpp (hunk 2 above)
Comment (zhoonit) [reply test/unittest/unittest_nntrainer_models.cpp:1228]: Let me do it with resnet validation :)

File: test/unittest/unittest_nntrainer_models.cpp (hunk 1 above)
Comment (kparichay) [reply test/unittest/unittest_nntrainer_models.cpp:296]: alright.

File: test/unittest/unittest_nntrainer_models.cpp (hunk 2 above)
Comment (kparichay) [reply test/unittest/unittest_nntrainer_models.cpp:1228]: cool.

**Review by kparichay — APPROVED** [review]
LGTM

**Comment by kparichay** [comment]
@jijoongmoon Please don't directly merge PRs waiting for other PRs.

**Pull request merged**